application/models/User.php

```diff
@@ -996,6 +996,16 @@ public function isExpired()
         return $expired;
     }
 
+    /**
+     * Returns true if user is active
+     * @return boolean
+     */
+    public function isActive()
```

Comment:
Is this related to this PR?
I believe it is a good method, but asking if related to the solution as it is not called.

Reply:
I add it during devlopement but remove it after.
https://github.com/LimeSurvey/LimeSurvey/pull/3767/commits/1c0cf421b72bad9e7a66fc77c101ea36b0ae3f2f

You're right :)